api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -252,6 +262,20 @@ def __init__(
         num_workers: int = 0,
         soma_buffer_bytes: Optional[int] = None,
     ) -> None:
+        """
+        Construct a new ``ExperimentDataPipe``.
+
+        Args:
```

Comment:
Recommend removing the empty annotations, as they will likely mess up the docsite.

Reply:
done